src/modules/tests/tests.selector.ts

```diff
@@ -68,3 +68,9 @@ export const isEndToEndPracticeTest = (tests: TestsModel) : boolean =>
 export const getAllTestStatuses = (test: TestsModel): { [slotId: string]: TestStatus; } => {
   return test.testStatus;
 };
+
+export const getActivityCodeBySlotId = (testsModel: TestsModel, id: number): ActivityCode => {
+  if (testsModel && testsModel.startedTests && testsModel.startedTests[id]) {
```

Comment:
What happens here in the case where one of these conditions is false?

Reply:
Have changed to return `null` if doesn't meet these conditions